react-native-cli/index.js

```diff
@@ -65,6 +66,30 @@ function init(name) {
     process.exit(1);
   }
 
+  if (fs.existsSync(name)) {
+    prompt.start();
+
+    var property = {
+      name: 'yesno',
+      message: 'Directory ' + name + ' already exist. Continue?',
+      validator: /y[es]*|n[o]?/,
+      warning: 'Must respond yes or no',
+      default: 'no'
+    };
+
+    prompt.get(property, function (err, result) {
+      if (result.yesno === 'yes') {
```

Comment:
your regex allows `y` and `n`, but the rest of code requires `yes`. Maybe check `yesno[0] === 'y'`?